lib/openxml/drawingml/elements/srgb_color.rb

```diff
@@ -0,0 +1,14 @@
+module OpenXml
+  module DrawingML
+    module Elements
+      class SRGBColor < OpenXml::Docx::Elements::Element
+        namespace :a
+        tag :srgbClr
+        name "srgb_color"
```

Comment:
`tag` is what is put in the xml. `name` is how other classes figure out which class to refer to? In any case, most default to turning camelCase into snake_case, but this one doesn't follow the same sort of pattern, so name is specified directly.